File: 2862.py
a = int(input())
lista = []
for i in range(0, a):
    g = int(input())
    if g > 8000:
        lista += ['Mais de 8000!']
    else:
        lista += ['Inseto!']
for h in range(0, a):
    print(f'{lista[h]}')

File: 2862.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var a = Number(lines.shift())
let lista = []
for(var i = 0;i < a; i++){
    var g = Number(lines.shift())
    if(g > 8000){
        lista.push('Mais de 8000!')}
    else{
        lista.push('Inseto!')}}
for(var h = 0;h < a; h++){
    console.log(`${lista[h]}`)}
